.eslintrc.js

```diff
@@ -53,20 +60,40 @@ module.exports = {
       'error',
       { ImportDeclaration: { multiline: true } },
     ],
-    // require line break after operator
-    'operator-linebreak': ['error', 'after'],
+    'operator-linebreak': 'off',
```

Comment:
does not behave properly for ternary operators

Reply:
Yeah, I have a VSCode prettier extension enabled, which is part of the issue. I get an error from ESLint, then save the file and Prettier auto-formats it. I've found that often ESLint rules conflict with Prettier, and I'm unsure how to give priority to Prettier (do we drop ESLint altogether?) We can discuss next week :)

Reply:
Not sure this is the reason, but the documentation for the eslint plugin says that you should put `plugin:prettier/recommended` last. So maybe this is it?
This should configure `eslint-config-prettier` which disables all stylistic rules, but it needs to come _after_ the others that might enable them.

Reply:
To be done in another PR :-)